mlir/include/mlir/Dialect/Vector/Utils/VectorUtils.h

```diff
@@ -180,6 +180,29 @@ struct MaskableOpRewritePattern : OpRewritePattern<SourceOp> {
 /// are not linearizable.
 bool isLinearizableVector(VectorType type);
 
+/// Create a TransferReadOp from `source` with static shape `readShape`. If the
+/// vector type for the read is not the same as the type of `source`, then a
+/// mask is created on the read, if use of mask is specified or the bounds on a
+/// dimension are different.
+///
+/// `enableMasking` if false, the inBoundsVal values are set properly, based on
```

Comment:
We need to update the doc comments w.r.t `useInBoundsInsteadOfMasking`.

Reply:
Ops... Missed that one. Good catch. Thanks!